worker/snapshot_test.go

```diff
@@ -34,6 +34,8 @@ import (
 	"github.com/dgraph-io/dgraph/testutil"
 )
 
+const testSchema = "type Person { name: String }"
```

Comment:
let's declare this where you are using it

Reply:
done